ttnn/cpp/ttnn/operations/experimental/ccl/CMakeLists.txt

```
@@ -4,8 +4,25 @@ add_library(TTNN::Ops::Experimental::CCL ALIAS ttnn_op_experimental_ccl)
 target_precompile_headers(ttnn_op_experimental_ccl REUSE_FROM TT::CommonPCH)
 TT_ENABLE_UNITY_BUILD(ttnn_op_experimental_ccl)
 
+set_target_properties(
+    ttnn_op_experimental_ccl
+    PROPERTIES
+        VERIFY_INTERFACE_HEADER_SETS
+            FALSE
+)
+
 target_sources(
     ttnn_op_experimental_ccl
+    PUBLIC
+        FILE_SET headers
```

```suggestion
        FILE_SET kernels
```

We should use a name that represents what they are.